client/css/scroll.css

```diff
@@ -0,0 +1,44 @@
+#sections {
+  position: relative;
+  top: 0;
+  z-index: 90;
+
+  display: inline-block;
+  width: 25vw;
+}
+  
+.step {
+  margin-bottom: 200px;
+}
+  
+#vis {
+  position: sticky;
+  top: 60px;
+  z-index: 1;
+
+  display: block;
+  width: 50vw;
+
+  height: 60vh;
+  margin-left: 0;
+
+  background-color: #dddddd;
+}
+
+.hero {
+  display: grid;
```

Comment:
Why display grid? Use `display: flex;` since it's less tightly coupled with a grid layout and will behave closer to a regular block element.

Reply:
http://howtocenterincss.com/ is a good resource for determining how to vertically center elements.

Reply:
When I followed `display:flex` and the link, the divs didn't center. The text went on left and right sides.